runner.go

```diff
@@ -10,17 +13,25 @@ import (
 	"os"
 	"path/filepath"
 	"regexp"
+	"strings"
 	"sync"
 	"time"
 
 	dep "github.com/hashicorp/consul-template/dependency"
 	"github.com/hashicorp/consul-template/watch"
 	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/go-multierror"
 )
 
 // Regexp for invalid characters in keys
 var InvalidRegexp = regexp.MustCompile(`[^a-zA-Z0-9_]`)
 
+// Status is an internal struct that is responsible for marshaling and
+// unmarshaling JSON responses into keys.
+type Status struct {
```

Comment:
It's worth putting the source / destination info here just for operators. Then they can see it in the UI and know what is happening.